DDD.Sessionize/Sync/SyncService.cs

```diff
@@ -49,6 +49,15 @@ public static async Task Sync(ISessionizeApiClient apiClient, ITableStorageRepos
                 (dest, src) => new { dest, src }).ToArray();
             var editedPresenters = existingPresenters.Where(x => !x.dest.GetPresenter().DataEquals(x.src)).ToArray();
 
+            // Update presenter ids on sessions
```

Comment:
This was the cause of the nasty bug.

Any session that appeared in `editedPresenters` would have it's ID changed further below which would mean that the ID was lost.

This would cause some of the sessions to fail at notification to the logic app.

There was one session stuck in an infinite loop of always being edited because of the other change in this PR around making syncing destructive rather than just additive.